docs/protocol/index.md

```diff
@@ -2,133 +2,91 @@
 software to seamlessly deploy smart contracts which interact with the rest of the Cosmos
-[ecosystem](https://cosmos.network/ecosystem)!
-
-import ProjectValue from '../../src/components/ProjectValue.js';
-
-
-## The EVMOS Token
-
-The denomination used for staking, governance and gas consumption on the EVM is the EVMOS. The EVMOS provides the utility of: securing the Proof-of-Stake chain, token used for governance proposals, distribution of fees to validator and users, and as a mean of gas for running smart contracts on the EVM.
+[ecosystem](https://cosmos.network/ecosystem).
 
-Evmos uses [Atto](https://en.wikipedia.org/wiki/Atto-) EVMOS as the base denomination to maintain parity with Ethereum.
+<!-- TODO ## High-level Systems Design (Add visualization) -->
 
-1 evmos = 10<sup>18</sup> aevmos
-
-This matches Ethereum denomination of:
+## Cosmos SDK
 
-1 ETH = 10<sup>18</sup> wei
+Evmos enables the full composability and modularity of the [Cosmos SDK](https://docs.cosmos.network/).
+As a Cosmos chain, Evmos is a sovereign blockchain with its own native token,
+that can connect to other chains through IBC. It includes standard modules from the Cosmos SDK,
+that work side to side with Evmos-specific modules, build by the Evmos core development team.
+Check out the [list of modules](modules/index.md) to get an overview of what each module is responsible for.
 
-## Cosmos Coins
+## CometBFT & ABCI
 
-Accounts can own Cosmos coins in their balance, which are used for operations with other Cosmos and transactions. Examples of these are using the coins for staking, IBC transfers, governance deposits and EVM.
+[CometBFT](https://github.com/cometbft/cometbft) (a fork of [Tendermint Core](https://docs.tendermint.com/)) consists of two chief technical components: a blockchain consensus
+engine and a generic application interface. The consensus engine, called
+CometBFT, ensures that the same
+transactions are recorded on every machine in the same order. The application
+interface, called the [Application Blockchain Interface (ABCI)](https://docs.tendermint.com/master/spec/abci/),
+enables the transactions to be processed in any programming language.
 
-## EVM Tokens
+CometBFT has evolved to be a general-purpose blockchain consensus engine that
+can host arbitrary application states. Since CometBFT can replicate arbitrary
+applications, it can be used as a plug-and-play replacement for the consensus
+engines of other blockchains. Evmos is an example of an ABCI application
+replacing Ethereum's PoW via CometBFT's consensus engine.
 
-Evmos is compatible with ERC20 tokens and other non-fungible token standards (EIP721, EIP1155)
-that are natively supported by the EVM.
+Another example of a cryptocurrency application built on CometBFT is the Cosmos
+network. CometBFT can decompose the blockchain design by offering a very
+simple API (ie. the ABCI) between the application process and consensus process.
```

Comment:
for my personal reading taste, the word CometBFT occurs weirdly often in these sentences